addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -416,6 +416,12 @@
                     <label class="col-form-label col-md-3" for="o_link_dialog_url_input">URL or Email</label>
                     <div class="col-md-9">
                         <input type="text" name="url" class="form-control" id="o_link_dialog_url_input" required="required"/>
+                        <div class="text-muted o_strip_domain">
+                            <input type="checkbox" id="o_link_dialog_url_strip_domain" checked="" class="form-check-input"/>
```

Comment:
While you change line just on top, please can you adapt the syntax to checked="checked"; more odoo-frinedly